source/common/network/io_socket_handle_impl.cc

```diff
@@ -17,6 +21,8 @@ IoError::IoErrorCode IoSocketError::errorCode() const {
   case EAGAIN:
     // EAGAIN should use specific error ENVOY_ERROR_AGAIN.
     NOT_REACHED_GCOVR_EXCL_LINE;
+  case EBADF:
```

Comment:
I don't think this should ever happen in real code, so I don't think it's worth actually mapping this. Can we cover this with asserts or just fix the tests somehow if this is being hit?

Reply:
removed this enum value.